package.json

```diff
@@ -25,14 +25,14 @@
     "tsc": "lerna run tsc",
     "tsc:listEmittedFiles": "lerna run tsc:listEmittedFiles --stream --no-prefix",
     "package": "lerna run package",
-    "bootstrap": "lerna bootstrap --no-ci --force-local --ignore-prepublish && npm run tsc && npm run package",
-    "bootstrap-no-build": "lerna bootstrap --no-ci --force-local && npm run tsc",
-    "bootstrap-no-build-no-scripts": "lerna bootstrap --no-ci --force-local --ignore-scripts && npm run tsc",
-    "bootstrap-no-build-no-concurrency": "lerna bootstrap --no-ci --force-local --concurrency 1 && npm run tsc",
-    "bootstrap-no-build-quiet": "lerna bootstrap --no-ci --force-local --loglevel=error && npm run tsc",
-    "ci:bootstrap": "lerna bootstrap --no-ci --force-local",
+    "bootstrap": "lerna bootstrap --no-ci --force-local && npm run package",
+    "bootstrap-no-build": "lerna bootstrap --no-ci --force-local",
```

Comment:
I'm feeling vague unease over the use of the term 'build' here.    That's not probably your work, just reflection on my part.   

Just for clarity - "no build" means following this PR: 

Do not package/webpack/etc but *do* compile typescript.